### PR TITLE
Code Cleanup

### PR DESCRIPTION
**Pull request by fenryka**

### CODE REVIEW

File: core/src/main/kotlin/net/corda/core/serialization/CordaSerializationTransformRename.kt [hunk 1]
```diff
@@ -18,11 +18,11 @@ annotation class CordaSerializationTransformRenames(vararg val value: CordaSeria
 // TODO When we have class renaming update the docs
 /**
  * This annotation is used to mark a class has having had a property element. It is used by the
- * AMQP deserialiser to allow instances with different versions of the class on their Class Path
+ * AMQP deserializer to allow instances with different versions of the class on their Class Path
  * to successfully deserialize the object
```
Comment (rnicoll) [inline core/src/main/kotlin/net/corda/core/serialization/CordaSerializationTransformRename.kt]: Full stop please if we're cleaning up comments

File: node-api/src/main/kotlin/net/corda/nodeapi/internal/serialization/amqp/ArraySerializer.kt [hunk 2]
```diff
@@ -21,7 +21,7 @@ open class ArraySerializer(override val type: Type, factory: SerializerFactory)
     // id to generate it properly (it will always return [[[Ljava.lang.type -> type[][][]
     // for example).
     //
-    // We *need* to retain knowledge for AMQP deserialisation weather that lowest primitive
+    // We *need* to retain knowledge for AMQP deserialization weather that lowest primitive
     // was boxed or unboxed so just infer it recursively
```
Comment (rnicoll) [inline node-api/src/main/kotlin/net/corda/nodeapi/internal/serialization/amqp/ArraySerializer.kt]: Full stop

File: node-api/src/main/kotlin/net/corda/nodeapi/internal/serialization/amqp/EvolutionSerializer.kt [hunk 3]
```diff
@@ -10,7 +10,7 @@ import kotlin.reflect.full.findAnnotation
 import kotlin.reflect.jvm.javaType
 
 /**
- * Serializer for deserialising objects whose definition has changed since they
+ * Serializer for deserializing objects whose definition has changed since they
  * were serialised
```
Comment (rnicoll) [inline node-api/src/main/kotlin/net/corda/nodeapi/internal/serialization/amqp/EvolutionSerializer.kt]: Same

File: node-api/src/main/kotlin/net/corda/nodeapi/internal/serialization/amqp/EvolutionSerializer.kt [hunk 4]
```diff
@@ -38,7 +38,7 @@ class EvolutionSerializer(
 
     companion object {
         /**
-         * Unlike the generic deserialisation case where we need to locate the primary constructor
+         * Unlike the generic deserialization case where we need to locate the primary constructor
          * for the object (or our best guess) in the case of an object whose structure has changed
          * since serialisation we need to attempt to locate a constructor that we can use. I.e.
          * it's parameters match the serialised members and it will initialise any newly added
```
Comment (rnicoll) [inline node-api/src/main/kotlin/net/corda/nodeapi/internal/serialization/amqp/EvolutionSerializer.kt]: it's -> its

Also I'd rather not start a sentence "I.e.", can you expand it to "For example" please?

File: node-api/src/main/kotlin/net/corda/nodeapi/internal/serialization/amqp/EvolutionSerializer.kt [hunk 5]
```diff
@@ -77,6 +77,8 @@ class EvolutionSerializer(
          *  as it was serialised and the type descriptor of that type
          * @param new is the Serializer built for the Class as it exists now, not
          * how it was serialised and persisted.
+         * @param factory the [SerializerFactory] associated with the serialization
+         * context this serializer is being built for
```
Comment (rnicoll) [inline node-api/src/main/kotlin/net/corda/nodeapi/internal/serialization/amqp/EvolutionSerializer.kt]: Full stop please

File: node-api/src/main/kotlin/net/corda/nodeapi/internal/serialization/carpenter/Schema.kt [hunk 6]
```diff
@@ -111,7 +111,7 @@ class EnumSchema(
 }
 
 /**
- * Factory object used by the serialiser when building [Schema]s based
+ * Factory object used by the serializer when building [Schema]s based
  * on an AMQP schema
```
Comment (rnicoll) [inline node-api/src/main/kotlin/net/corda/nodeapi/internal/serialization/carpenter/Schema.kt]: Full stop :)

File: core/src/main/kotlin/net/corda/core/serialization/CordaSerializationTransformRename.kt (hunk 1 above)
Comment (fenryka) [reply core/src/main/kotlin/net/corda/core/serialization/CordaSerializationTransformRename.kt]: done

File: node-api/src/main/kotlin/net/corda/nodeapi/internal/serialization/amqp/ArraySerializer.kt (hunk 2 above)
Comment (fenryka) [reply node-api/src/main/kotlin/net/corda/nodeapi/internal/serialization/amqp/ArraySerializer.kt]: done

File: node-api/src/main/kotlin/net/corda/nodeapi/internal/serialization/amqp/EvolutionSerializer.kt (hunk 3 above)
Comment (fenryka) [reply node-api/src/main/kotlin/net/corda/nodeapi/internal/serialization/amqp/EvolutionSerializer.kt]: done

File: node-api/src/main/kotlin/net/corda/nodeapi/internal/serialization/amqp/EvolutionSerializer.kt (hunk 5 above)
Comment (fenryka) [reply node-api/src/main/kotlin/net/corda/nodeapi/internal/serialization/amqp/EvolutionSerializer.kt]: done

**Comment by fenryka** [comment]
@rnicoll - changes made

**Pull request merged**